scala-thoughts.md

```diff
@@ -0,0 +1,2 @@
+## My thoughts on Scala
+_I actually think that the scala interface is pretty easy. The only thing that I think I would have trouble with is the testing. The structure and syntax for testing don't really make sense to me. I don't really understand how should be works, and why it would work even when there's a space between the should be and parentheses. Everything else is pretty similar to Python. Not having return statements is strange to me, but having a return type declared at the top of the function makes up for that confusion. I like the style of declaring parameters' types better than Java's, but I don't like using the same format for declaring the types of variables. I don't think scala needs to have both val and var as keywords, I think vars could just exist without vals. I had trouble before I found out that vals weren't able to be changed. Overall, I enjoyed learning Scala because of its similarity to other languages._
```

Comment:
I agree with a lot of these thoughts on Scala, I really liked how simple it looks but it is very odd having no return keyword and trying to differentiate between val and var. I really liked how sleek the look of the tests is but I do think it is a bit difficult to implement for the first time so I think that is something I need to practice more in Scala.